cmd/zq/zq.go

```diff
@@ -27,9 +27,35 @@ func (reason errInvalidFile) Error() string {
 
 var Zq = &charm.Spec{
 	Name:  "zq",
-	Usage: "zq [options] <search> [file...]",
+	Usage: "zq [ options ] [ zql ] file [ file ... ]",
 	Short: "command line zeek processor",
-	Long:  "",
+	Long: `
+zq is a command-line tool for processing logs.  It applies boolean logic
+to filter each log value, optionally computes analytics and transformations,
+and writes the output to one or more files or standard output.
+
+The input  and output formats are either specified explicitly or derived from
+file name extensions.  Supported input formats include ZSON (.zson), JSON (.json),
+NDJSON (.ndjson), and zeek log format (.log).  Supported output formats include
+all the input formats along with text and tabular formats.
+
+zq must be run with at least one input file specified.  As with awk, standard input
+can be specified with a "-" in the place of the file name.  Output is sent to
+standard output unless a -o or -d argument is provided, in which case output is
+sent to the indicated file comforming to the type implied by the extension (unless
+-f explicitly indicates the output type).
+
+After the options, the query may be specified as a
+single argument conforming with zql syntax, i.e., it should be quoted as
+a single string in the shell.
+If the first argument is a path to a valid file rather than a zql query,
+then the zql expression is assumed to be "*", i.e., match and output all
+of the input.  If the first argument is both valid zql and an existing file,
+then the file overrides.
```

Comment:
s/zql/ZQL/g

Reply:
I like `zql` better.

Reply:
I like zql better too but to be consistent with Phil's conventions, I think we go with ZQL